### PR TITLE
Add minimal documentation for `EXPLAIN`

### PR DESCRIPTION
**Pull request by vogelsgesang**

### CODE REVIEW

File: website/docs/sql/command/explain.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+:::warning Unstable Interface
+The returned query plans map directly to Hyper's internal represenation of query plans.
+As Hyper evolves, the query plans might change considerably.
+Don't expect query plans (neither their formatting nor their semantics) to be stable across Hyper versions.
+:::
+
+
+This command retrieves the query plan which Hyper uses to execute the given query.
+The query plan describes in which tables are accessed in which order, how they are joined together, which expressions are evaluated and all other details of query execution.
```
Comment (haubenmi) [inline website/docs/sql/command/explain.md:22]:
```suggestion
The query plan describes which tables are accessed in which order, how they are joined together, which expressions are evaluated and all other details of query execution.
```

File: website/docs/sql/command/explain.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+One or more `<option>`s can provided to change which information is displayed and how it is displayed.
```
Comment (haubenmi) [inline website/docs/sql/command/explain.md:24]:
```suggestion
One or more `<option>`s can be provided to change which information is displayed and how it is displayed.
```

File: website/docs/sql/command/explain.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+There are three types of plans which can be queried:
+
+* The *optimized plan*. By default, if no other behavior is requested through an `<option>`, `EXPLAIN` will display the optimized plan.
+* The *optimizer steps*. If the `OPTIMIZERSTEPS` option is present, Hyper will output the plan at multiple intermediate steps during query optimization, e.g., before and after join reordering.
+* The *analyzed plan*. When invoked with the `ANALYZE` option, Hyper will actually execute the query, including all side effects (inserted/deleted tuples, etc.). Instead of the normal query results, you will however receive the query plan of the query, annotated with runtime statistics such as the number of tuples processed by each operator.
+
+By default, plans are formatted as ASCII art.
+The ASCII art is useful for quickly grasping the overall query structure, but is lacking details, such as the exact join conditions.
+Those details are present in the JSON output format, which can be requested using the `VERBOSE` option. 
+The `VERBOSE` option can be combined with any other choice
```
Comment (haubenmi) [inline website/docs/sql/command/explain.md:37]:
```suggestion
The `VERBOSE` option can be combined with any other choice.
```

File: website/docs/sql/command/explain.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+To visualize a JSON query plan and interactively explore it, copy-paste the plan into [Hyper's Plan Viewer](https://tableau.github.io/query-graphs/).
+
+## Examples
+
+The following will give us the query plan of TPC-H query 6, formatted as an ASCII art.
```
Comment (haubenmi) [inline website/docs/sql/command/explain.md:42]:
```suggestion
The following will give us the query plan of TPC-H query 6, formatted as ASCII art.
```

File: website/docs/sql/command/explain.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+If we want to view the plan in a graphical plan viewer, or want to see additional details in the plan, we can use the `VERBOSE` option and then copy-paste the resulting JSON into [Hyper's Plan Viewer](https://tableau.github.io/query-graphs/).
+
+```
+SELECT sum(l_extendedprice * l_discount) as revenue
+FROM lineitem
+WHERE l_shipdate >= date '1994-01-01'
+    AND l_shipdate < date '1995-01-01'
+    AND l_discount BETWEEN 0.06 - 0.01 AND 0.06 + 0.01
+    AND l_quantity < 24
+```
```
Comment (haubenmi) [inline website/docs/sql/command/explain.md:75]: Is this missing EXPLAIN VERBOSE?

File: website/docs/sql/command/explain.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
+# EXPLAIN
+
+— show the query plan for a query
+
+## Synopsis
+
+```sql_template
+EXPLAIN (<option>, ...) <query>;
```
Comment (hofst) [inline website/docs/sql/command/explain.md:8]: maybe I missed it, but I think we should list the available options here.

**Review by hofst — APPROVED** [review]

**Pull request merged**